### ISSUE TITLE
Move _* attributes to the top

### ISSUE THREAD

**Issue by mmuehlfeldRH** [redhat-documentation/newdoc#42]
Some of the _* attributes are separated with a blank line from each other or appear later in the template. For consistency reasons, can we move them up so that they appear all at the beginning of the file (without blank lines)

![image](https://github.com/redhat-documentation/newdoc/assets/43061675/02e97c94-901b-4691-a35e-f46619ca7dcf)

**Comment by msuchane** [comment]
I agree it would be prettier to move the attributes together. The reason why the mod docs attribute is placed further down is that it follows the attribute placement in the upstream template files:

https://github.com/redhat-documentation/modular-docs/tree/main/modular-docs-manual/files

I'll see if I can consolidate this without diverging from the templates too much.

**Comment by mmuehlfeldRH** [comment]
Then it should better be improved upstream, so that it won't come back after the next template refresh.

**Comment by mmuehlfeldRH** [comment]
Solved by https://github.com/redhat-documentation/newdoc/pull/47
